taxonomies/food/categories.txt

```diff
@@ -74013,6 +74022,10 @@ ciqual_food_code:en: 11015
 ciqual_food_name:en: Black pepper, powder
 ciqual_food_name:fr: Poivre noir, poudre
 
+< en:Ground black peppers
+en: Coarsely ground black peppers
```

Comment:
We are in the process of removing ingredients processing from ingredients.txt, could you add "coarsely ground" as a processing in ingredients_processing.txt indeed? (maybe as a synonym of "ground", it may not be very useful to distinguish "coarsely ground" from "ground". Or as a separate child entry of "ground".

Reply:
Ah sorry just saw this is categories.txt and not ingredients.txt, ignore my previous comment :)